opent2t.p.thermostat/com.wink.thermostat/js/tests/test.js

```diff
@@ -0,0 +1,224 @@
+var test = require('ava');
+var OpenT2T = require('opent2t').OpenT2T;
+var config = require('./testConfig');
+var http = require('http');
+var q = require('q');
+
+console.log("Config:");
+console.log(JSON.stringify(config, null, 2));
+
+var translatorPath = require('path').join(__dirname, '..');
+var hubPath = require('path').join(__dirname, '../../../../org.opent2t.sample.hub.superpopular/com.wink.hub/js');
+
+var translator = undefined;
+var deviceInfo = {};
+
+function getThermostat(devices) {
+    for (var i = 0; i < devices.length; i++) {
+        var d = devices[i];
+
+        if (d.openT2T.translator === 'opent2t-translator-com-wink-thermostat') {
+            return d;
+        }
+    }
+
+    return undefined;
+}
+
+// setup the translator before all the tests run
+test.before(async () => {
+    var hubTranslator = await OpenT2T.createTranslatorAsync(hubPath, 'thingTranslator', {'accessToken': config.accessToken});
+    var hubInfo = await OpenT2T.getPropertyAsync(hubTranslator, 'org.opent2t.sample.hub.superpopular', 'HubResURI');
```

Comment:
>getPropertyAsync [](start = 32, length = 16)

New hub uses invokeMethodAsync for getPlatforms rather than getpropertyAsync and the resulting hubInfo contains platforms rather than devices.